### PR TITLE
add font-awesome-class to show icon with new bootstrap

### PR DESCRIPTION
**Pull request by stefina**
this works with the current bootstrap and the older bootstrap versions as well.

### CODE REVIEW

File: ckanext/showcase/templates/showcase/search.html [hunk 1]
```diff
@@ -48,6 +48,6 @@
       {{ h.snippet('snippets/facet_list.html', title=c.facet_titles[facet], name=facet) }}
     {% endfor %}
   </div>
-  <a class="close no-text hide-filters"><i class="icon-remove-sign"></i><span class="text">close</span></a>
+  <a class="close no-text hide-filters"><i class="fa fa-times icon-remove-sign"></i><span class="text">close</span></a>
```
Comment (metaodi) [inline ckanext/showcase/templates/showcase/search.html]: This should be `fa-times-circle`

**Comment by metaodi** [comment]
I opened an issue in CKAN core about the renaming of the routes: https://github.com/ckan/ckan/issues/4066

**Comment by stefina** [comment]
Thank you @metaodi . As the build only fails for master and as far as I checked there are no other reasons for failing other than the renaming routes, I would merge this, ignoring that the build fails against current master. Unless you have other concerns?

**Comment by metaodi** [comment]
All good.

**Pull request merged**